src/assets/HTMLAsset.js

```diff
@@ -41,6 +41,28 @@ class HTMLAsset extends Asset {
     this.ast.walk(node => {
       if (node.attrs) {
         for (let attr in node.attrs) {
+          if (node.tag === 'img' && attr === 'srcset') {
+            const newSources = [];
+            for (const source of node.attrs[attr].split(',')) {
+              const pair = source.trim().split(' ');
+              if (pair.length === 0) continue;
+
+              let assetPath = this.addURLDependency(
+                decodeURIComponent(pair[0])
+              );
+              if (!isURL(assetPath)) {
+                assetPath = urlJoin(this.options.publicURL, assetPath);
+              }
+              pair[0] = assetPath;
```

Comment:
The logic of the above few lines could be shared with the copy below which was already there. Maybe make a function for it?

Reply:
Done.